src/main/java/com/netuitive/ananke/statsd/client/request/DecrementRequest.java

```diff
@@ -12,7 +12,7 @@
     public DecrementRequest() {
     }
 
-    public DecrementRequest(String metric, Long value, List<Tag> tags, Long sampleRate) {
+    public DecrementRequest(String metric, Double value, List<Tag> tags, Long sampleRate) {
```

Comment:
What about backwards compatibility for this too? I would expect there to be an overloaded constructor such as:
```
public DecrementRequest(String metric, Long value, List<Tag> tags, Long sampleRate) {
    this(metric, value == null ? null : value.doubleValue(), tags, sampleRate);
}
```